src/md/release.md

```diff
@@ -156,6 +156,14 @@ svn import svn-dist https://dist.apache.org/repos/dist/dev/apex/${RNAME}-RC1 -m
 
 ## Build and Deploy Documentation
 
+### Javadoc
```

Comment:
@vrozov The javadoc is built from the release branch, that was the intention so that we have the latest patches. Otherwise you would need to build it from the tag. Since you are at it, can you merge the PR?